### PR TITLE
El 1339 parallel specs

### PR DESCRIPTION
**Pull request by starswan**
https://dsdmoj.atlassian.net/browse/EL-1339

## What changed and why

Spike for parallel tests gem

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing
- Branch is generally up to date with main Github - definitely no conflicts
- No unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- PR description says *what* changed and *why*, with a link to the JIRA story.
- Diff has been checked for unexpected changes being included.
- Commit messages say why the change was made.

### CODE REVIEW

File: .rspec_parallel [hunk 1]
```diff
@@ -0,0 +1,6 @@
+--require spec_helper
```
Comment (MazOneTwoOne) [inline .rspec_parallel:1]: L&D question - I don't quite understand how this get's invoked or why it's needed. It is mainly for formatting, helping with logs and tagging? Can't make sense of [the documentation](https://github.com/grosser/parallel_tests?tab=readme-ov-file#loggers).  

My only other thought is that this set-up help with the coverage report?

File: spec/system/feedback_spec.rb [hunk 2]
```diff
@@ -23,12 +23,18 @@
         click_on "Continue to download and finish"
         expect(page).to have_content("Were you satisfied with this service?")
         click_on "Yes"
+        # TODO:  temp fix for parallel running
```
Comment (MazOneTwoOne) [inline spec/system/feedback_spec.rb:26]: Question - is this a temp fix for when we get to [EL-1374](https://dsdmoj.atlassian.net/browse/EL-1374)?

**Review by MazOneTwoOne — APPROVED** [review]
Looks good. I'll update Jira ticket with more feedback as this is a spike. 

Also had one L&D question.

File: .rspec_parallel (hunk 1 above)
Comment (starswan) [reply .rspec_parallel]: This isn't actually required as all the tests require 'rails_helper' which includes this. I've taken it out. Its default for rspec, which is not rails-specific

File: spec/system/feedback_spec.rb (hunk 2 above)
Comment (starswan) [reply spec/system/feedback_spec.rb:26]: Yes, this is currently a problem anyway, but when running lots of processes (10 x browsers and 10 rspecs on my M1 machine by default) it happens so often that a clean build is almost impossible to create

**Pull request merged**